Apps/Common/src/Delegates/AuthServer/Keycloak/UserDelegate.cs

```diff
@@ -0,0 +1,196 @@
+//-------------------------------------------------------------------------
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Common.Delegates.AuthServer.Keycloak
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Diagnostics.Contracts;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Net.Mime;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+
+    using HealthGateway.Common.Delegates.AuthServer;
+    using HealthGateway.Common.Models.AuthServer;
+    using HealthGateway.Common.Services;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+
+
+
+
+    /// <summary>
+    /// Class implementation of <see cref="IUserDelegate"/> that uses http REST to connect to Keycloak.
+    /// </summary>
+    public class UserDelegate : IUserDelegate
```

Comment:
Is this a KeyCloakUserDelegate?

Reply:
yes, it is, but the entire class is scoped by the namespace.